.github/workflows/build-test.yml

```diff
@@ -29,6 +29,8 @@ jobs:
           cache: 'npm'
           cache-dependency-path: '**/package-lock.json'
       - run: npm ci
+      - name: Install Playwright
```

Comment:
I don't disagree with doing this, however it does slow down the CI with this installation step, I kind of dislike that part. The tests themselves do look to be running faster in playwright so it might be a win in the end